packages/styles/framework/src/utilities/mixins/_mixins.scss

```
@@ -155,20 +211,48 @@
     }
   }
 
+  .#{$type}-tiles:not(.is-grid) {
+    .#{$type}-tile:not(.is-horizontal) {
+      max-width: 160px;
+    }
+  }
+
   //TILE ONLY
   .#{$type}-tile {
     position: relative;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "position" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)